app/package.json

```diff
@@ -170,6 +170,7 @@
     "copyfiles": "^2.4.1",
     "cross-env": "^7.0.3",
     "dotenv": "^16.4.5",
+    "electron-browser-storage": "^1.0.7",
```

Comment:
I'd prefer not to add another dependency for this. I prefer in the end to save it using a file as you had suggested than introducing another dependency. My main problem with a preferences file is that then we have 2 files related to preferences. Maybe we actually remove the default one as a clean up action when we quit the app (assuming Electron has no way to prevent this by default)...